agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/BaseTracer.java

```diff
@@ -127,7 +131,47 @@ public void addThrowable(final Span span, final Throwable throwable) {
     span.recordException(throwable);
   }
 
-  public static void setPeer(final Span span, String peerName, String peerIp) {
-    BaseDecorator.setPeer(span, peerName, peerIp);
+  public void onPeerConnection(final Span span, final InetSocketAddress remoteConnection) {
```

Comment:
can you update the corresponding methods in `BaseDecorator` to delegate to these methods?

and the same with the methods copied from `HttpClientDecorator` to `HttpClientTracer` where possible?

Reply:
Updated.